.travis.yml

```diff
@@ -1,16 +1,28 @@
-language: go
+---
 
-matrix:
-  include:
-  -  os: linux
-     sudo: required
-     services:
-       - docker
-  -  os: osx
```

Comment:
What’s all of this? Why is this a part of this PR, when the CI does not use the Nix build at all?

Reply:
Originally I hope to build static binary with `.travis.yml`, too; BTW it take more than 50mins due to Travis CI's limited CPU (x2) so finally I remove the test.

It is now rolling back as original `.travis.yml`